src/urllib3/contrib/securetransport.py

```diff
@@ -188,6 +186,7 @@ def inject_into_urllib3():
     util.ssl_.HAS_SNI = HAS_SNI
     util.IS_SECURETRANSPORT = True
     util.ssl_.IS_SECURETRANSPORT = True
+    util.ssl_.USE_SYSTEM_SSL_CIPHERS = False
```

Comment:
For now we always want to set_ciphers() on SecureTransport pending investigation in #2091

Reply:
in #2091 we decided that SecureTransport ciphers were fine, so we can remove a lot of code now!